docs/admin/authorization/rbac.md

```diff
@@ -41,15 +42,19 @@ metadata:
   namespace: default
   name: pod-reader
 rules:
-  - apiGroups: [""] # The API group "" indicates the core API Group.
-    resources: ["pods"]
-    verbs: ["get", "watch", "list"]
+- apiGroups: [""] # The API group "" indicates the core API Group.
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
 ```
 
-`ClusterRoles` hold the same information as a `Role` but can apply to any
-namespace as well as non-namespaced resources (such as `Nodes`,
-`PersistentVolume`, etc.). The following `ClusterRole` can grant permissions to
-read secrets in any namespace.
+`ClusterRole` objects can be used to grant the same permissions as `Role` objects,
+but because they are cluster-scoped, they can also be used to grant access to:
+
+* cluster-scoped resources (like nodes)
+* non-resource endpoints (like `/healthz`)
+* namespaced resources (like pods) across all namespaces (needed to run `kubectl get pods --all-namespaces`, for example)
+
+The following `ClusterRole` can be used to grant read access to secrets in any namespace, or across all namespaces:
```

Comment:
If we take @deads2k's suggestion there, I'd like to see one transitional sentence after the example YAML figure like the following:
> A `ClusterRole`'s scope of influence is as broad as cluster-wide or as narrow as a particular namespace, depending on what kind of _binding_—introduced in the next section—references it.